spec/requests/subscription/tiered_membership_spec.rb

```diff
@@ -325,4 +325,81 @@
       end
     end
   end
+
+  context "when product has custom fields" do
+    before :each do
+      @product.custom_fields.create!(
+        name: "Favorite Color",
+        required: false,
+        field_type: "text",
+        seller_id: @product.user.id
+      )
+      @product.custom_fields.create!(
+        name: "Subscribe to Newsletter",
+        required: false,
+        field_type: "checkbox",
+        seller_id: @product.user.id
+      )
+    end
+
+    it "does not display custom fields on the manage subscription page" do
```

Comment:
I feel like all three of these tests could be combined; they naturally follow on from each other. Could you see if that works? (would save a few seconds on CI, which adds up over many PRs!)